Formula/jiq.rb

```diff
@@ -0,0 +1,19 @@
+class Jiq < Formula
+  desc "Jid on jq - interactive JSON query tool using jq expressions"
+  homepage "https://github.com/fiatjaf/jiq/"
+  url "https://github.com/fiatjaf/jiq/archive/0.7.1.tar.gz"
+  sha256 "fdb4a16d6965bbe5f5d053c14d75d221d90dded82f57db2134c5861bf3b8ee3e"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "jq"
+
+  def install
+    system "go", "build", "-o", bin/"jiq", "cmd/jiq/jiq.go"
+    prefix.install_metafiles
+  end
+
+  test do
+    system "true"
```

Comment:
Can you propose a proper test, and one more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
It's not going to be easy because the thing is an interactive program for the terminal. All the tests I can think of are either actually tests of `jq` or require knowledge of terminal-driving stuff that I don't have, usually both.

Reply:
Also, `jid`, already packaged, which has somewhat similar functionality, has similar tests.

Reply:
It does indeed, but it shouldn't. The correct thing to do about formulae with bad tests isn't to create more formulae with bad tests.

Reply:
Would you be so kind as to point me to a formula that does a similar thing (interactive terminal application) and has tests you like better?

Reply:
Try looking at any of the formulae that have `require "pty"` in their test blocks; one of them might be useful.